src/main/scala/kinesis/mock/cache/CacheConfig.scala

```diff
@@ -31,6 +35,27 @@ final case class CacheConfig(
 object CacheConfig {
   implicit val cacheConfigReader: ConfigReader[CacheConfig] = deriveReader
   implicit val cacheConfigCirceEncoder: Encoder[CacheConfig] = deriveEncoder
+
+  implicit val initialStreamsReader: ConfigReader[List[CreateStreamRequest]] = {
+    ConfigReader.fromString { s =>
+      s.split(',')
```

Comment:
Perhaps we should just use a Json Array here with JSON objects. It would make parsing a bit easier.

Reply:
I agree JSON object is easier to parse, but it's tougher to write a JSON string into an environment variable, especially if it requires multiple lines.

I thought this would've been a good middle ground, but if you're not too fond of this parsing, another option I can think of is to read it from a JSON file.

Reply:
We can keep this parsing, I'm not terribly opposed to it.